packages/sync-actions/src/projects.js

```diff
@@ -16,6 +20,14 @@ function createChannelsMapActions(mapActionGroup, syncActionConfig) {
       )
     )
 
+    allActions.push(
+      mapActionGroup('base', () => actionsMapBusinessUnit(diff, oldObj, newObj))
+    )
+
+    allActions.push(
+      mapActionGroup('base', () => actionsMapCustomer(diff, oldObj, newObj))
+    )
```

Comment:
i would suggest, to add two new ActionGroups ex: businessUnit.

`base` is mainly used for 1st lvl attributes ex: name, countries, etc..
plus `packages/sync-actions/src/projects-actions.js` shows that you already naming it perfectly diff than `base`

`actionsMapCustomer`: if im not mistaken thats more of generating the search config actions, so maybe more descriptive naming

Reply:
Hi @markus-azer , thanks for the feedback! I split the actions into three groups accordingly and renamed `actionsMapCustomer` to be more descriptive.